examples/connect-to-Redshift-using-pyspark-scala-sparkr.ipynb

```diff
@@ -0,0 +1,204 @@
+    "* This example we connect to Amazon Redshift cluster, hence the EMR cluster attached to this notebook must have the connectivity (VPC) and appropriate rules (Security Group).\n",
+    "\n",
+    "\n",
+    "## Introduction\n",
+    "In this example we use Pyspark, Spark Scala, and Spark R to connect to a table in Amazon Redshift using spark-redshift connector.\n",
+    "\n",
+    "[spark-redshift](#https://github.com/spark-redshift-community/spark-redshift) is a performant Amazon Redshift data source for Apache Spark\n",
+    "\n",
+    "## Setup\n",
+    "\n",
+    "* Create an S3 bucket location to be used as a temporary location for Redshift dataset. For example: s3://EXAMPLE-BUCKET/temporary-redshift-dataset/\n",
+    "\n",
+    "* Create an AWS IAM role which will be associated to the Amazon Redshift cluster. Make sure that this IAM role has access to read and write to the above mentioned S3 bucket location with the appropriate IAM policy. More details:\n",
+    "\n",
+    " * [Create AWS IAM role for Amazon Redshift](#https://docs.aws.amazon.com/redshift/latest/gsg/rs-gsg-create-an-iam-role.html)\n",
+    " * [Associate IAM role with Amazon Redshift cluster](#https://docs.aws.amazon.com/redshift/latest/dg/c-getting-started-using-spectrum-add-role.html)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2bcd6581",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%configure -f\n",
+    "{ \n",
+    "    \"conf\": \n",
+    "        {\n",
+    "            \"spark.jars.packages\": \"org.apache.spark:spark-avro_2.11:2.4.2,io.github.spark-redshift-community:spark-redshift_2.11:4.0.1\"\n",
+    "        }\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e4b48cb",
+   "metadata": {},
+   "source": [
+    "## Connect to Amazon Redshift using Pyspark"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9a068740",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%pyspark\n",
+    "\n",
+    "#Declare the variables and replace the variables values as appropiate\n",
+    "\n",
+    "str_jdbc_url='jdbc:redshift://<Redshift Endpoint>:5439/dev?user=<username>&password=<password>'\n",
+    "str_dbname='<Database name>'\n",
+    "str_tgt_table='<Target Table>'\n",
+    "str_s3_path='s3://<S3 Path>'\n",
+    "str_iam_role='<IAM Role ARN>'\n",
+    "\n",
+    "# Read data from source table\n",
+    "\n",
+    "jdbcDF = spark.read \\\n",
+    "    .format(\"io.github.spark_redshift_community.spark.redshift\") \\\n",
+    "    .option(\"url\", str_jdbc_url) \\\n",
+    "    .option(\"dbtable\", str_dbname) \\\n",
+    "    .option(\"tempdir\", str_s3_path) \\\n",
+    "    .option(\"aws_iam_role\",str_iam_role) \\\n",
+    "    .load()\n",
+    "\n",
+    "jdbcDF.limit(5).show()\n",
+    "\n",
+    "# Read data from target table\n",
```

Comment:
This should be write

Reply:
This has been fixed. Pull request updated. Thanks